api/src/main/kotlin/edu/wgu/osmt/elasticsearch/FindsAllByPublishStatus.kt

```diff
@@ -1,43 +1,69 @@
 package edu.wgu.osmt.elasticsearch
 
+import co.elastic.clients.elasticsearch._types.query_dsl.QueryBuilders.matchAll
+import edu.wgu.osmt.collection.CollectionDoc
 import edu.wgu.osmt.db.PublishStatus
-import org.elasticsearch.index.query.BoolQueryBuilder
-import org.elasticsearch.index.query.QueryBuilders
+import edu.wgu.osmt.elasticsearch.WguQueryHelper.createMatchPhrasePrefixDslQuery
```

Comment:
I would discourage using WGU in a class name. If you need to distinguish between elasticsearch and this code then use OSMT to clarify.

Reply:
Thx for catching that.